client/src/config.js

```diff
@@ -1,9 +1,9 @@
 const config = {
   // OAuth info (copied from the FusionAuth admin panel)
-  clientID: '48f671cc-1627-41b5-aed1-260b80a9f742',
-  clientSecret: 'pqQj3uOj7q7BLe0ImvgSxCvKUTuczAth1vjAmkntve8',
+  clientID: '85a03867-dccf-4882-adde-1a79aeec50df',
+  clientSecret: '6HvlAe28V4PL9OEKJ0VlZ6ZsEAluEsZHVFYKuoLAr1U',
```

Comment:
You are going to have to modify the kickstart file to specify the clientSecret.

https://github.com/FusionAuth/fusionauth-example-react-2.0/blob/main/kickstart/kickstart.json#L18

Reply:
application.oauthConfiguration.clientSecret as defined here: https://fusionauth.io/docs/v1/tech/apis/applications#create-an-application

Reply:
I'd rather not go messing around in that repo, so I changed the language to just tell them to copy the values over from the FusionAuth admin screen to the config file.

Reply:
I think we should do this in kickstart since that is the purpose of that feature. That way the user does not have to do this manually. 

You could either hard code it hear, and then also add it to the kickstart.json file ( https://github.com/FusionAuth/fusionauth-example-react-2.0/blob/da927b3e091f55a00608e83f712348f2dd4dee4b/kickstart/kickstart.json#L13 ) or generate it in an init script so it is different for each user, and then set it on the `kickstart` so that the Application gets created with the correct value. 

We should be able to know the `applicationId`, `clientId` and `clientSecret` and sync those up between this config and the kickstart.

Reply:
Added clientSecret to kickstart/kickstart.json, and amended README to mention it.